tests/test_oxli_functions.py

```diff
@@ -0,0 +1,42 @@
+from __future__ import print_function
+from __future__ import absolute_import
+from __future__ import unicode_literals
+#
+# This file is part of khmer, https://github.com/dib-lab/khmer/, and is
+# Copyright (C) Michigan State University, 2009-2015. It is licensed under
+# the three-clause BSD license; see LICENSE.
+# Contact: khmer-project@idyll.org
+#
+
+# pylint: disable=C0111,C0103,E1103,W0612
+
+from . import khmer_tst_utils as utils
+import khmer
+from oxli import functions
+
+
+def test_estimate_functions_1():
+    res = functions.estimate_optimal_with_N_and_M(99, 1024)
+    assert res[0] == 7, res[0]
+    assert res[1] == 146, res[1]
+    assert res[2] == 1022, res[2]
+    assert abs(.008 - res[3]) < .001, res[3]
```

Comment:
What do you think about exploring namedtuples for return values? I see these tests and I don't know what res[0] means, I need to go back to the function and what it returns. Namedtuples will continue working with this test, but we can also ask for `res.intZ` and have the same result.
(and, in this case, maybe some names better than `intZ, H1, M1, f1` would be nice).

Suggestion: keep these tests, modify the code to return a namedtuples, and add the namedtuple tests here (but don't delete the current ones).

This doesn't prevent the merge, but I do think it is a nice playground to modify how we return values from functions in our codebase.

[reference][1]

[1]: http://docs.quantifiedcode.com/python-code-patterns/readability/not_using_named_tuples_when_returning_more_than_one_value.html